src/config.json

```diff
@@ -16,5 +16,6 @@
   "twitterCard": "summary",
   "twitterImgAlt": "The Node.js Hexagon Logo",
   "facebook": "https://www.facebook.com/nodejsfoundation/",
-  "bannerBtnText": "Blog post"
+  "bannerBtnText": "Blog post",
+  "nodeGcalId": "nodejs.org_nr77ama8p7d7f9ajrpnu506c98@group.calendar.google.com"
```

Comment:
Can this be hardcoded? @joesepi @MylesBorins

Reply:
It can be. I wasn’t sure if it would change or not so I added it to the config